scripts/reports/bundle-size/command.js

```diff
@@ -21,9 +21,50 @@ async function buildFiles() {
 
 function readFileSizes() {
   console.log('getting file sizes');
-  const path = resolve('packages/headless/.size-snapshot.json')
-  const buffer = readFileSync(path)
-  return JSON.parse(buffer.toString());
+  const dir = resolve(__dirname, '../../../packages/headless/dist/browser');
+  const entries = getUseCasesAndFilePaths(dir);
+
+  return entries
+    .map((entry) => {
+      const {useCase, filePath} = entry;
+      const {size} = statSync(filePath);
+      return {useCase, size};
+    })
+    .reduce((acc, curr) => ({...acc, [curr.useCase]: curr.size}), {});
```

Comment:
I think this would be easier to read using `Object.fromEntries`

Reply:
Thanks! TIL about this method,